### DISCUSSION TITLE
Welcome to Accessible Astro Starter Discussions!

### DISCUSSION THREAD

**Discussion by markteekman** [incluud/accessible-astro-starter#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cf2013** [comment]
hi, great work, Any recommendations about finding guides or principles to make this work on azure static pages?

**Reply by markteekman** [reply]
@cf2013 I myself don't have any experience using this with Azure static pages, have you tried asking around on the official Astro Discord server?
